Assets/HoloToolkit/Dialogs/Scripts/Fitbox.cs

```diff
@@ -0,0 +1,109 @@
+        public bool MoveCollectionOnDismiss = false;
+        [Tooltip("The scene object to activate and reposition.")]
+        public GameObject StartupObject;
+        [Tooltip("The distance at which you want the Fitbox, and subsequent content display in front of the user.")]
+        public float Distance = 1.0f;
+        private Interpolator interpolator;
+        // The offset from the Camera to the StartupObject when
+        // the app starts up. This is used to place the StartupObject
+        // in the correct relative position after the Fitbox is
+        // dismissed.
+        private Vector3 collectionStartingOffsetFromCamera;
+
+        private void Start()
+        {
+            // This is the object to show when the Fitbox is dismissed
+            if (StartupObject != null)
+            {
+                collectionStartingOffsetFromCamera = StartupObject.transform.localPosition;
+                StartupObject.SetActive(false);
+            }
+            interpolator = GetComponent<Interpolator>();
+            interpolator.PositionPerSecond = 2f;
+        }
+
+        void LateUpdate()
+        {
+            Transform cameraTransform = Camera.main.transform;
+
+            interpolator.SetTargetPosition(cameraTransform.position + (cameraTransform.forward * Distance));
+            interpolator.SetTargetRotation(Quaternion.LookRotation(-cameraTransform.forward, -cameraTransform.up));
+        }
+
+        public void OnInputClicked(InputClickedEventData eventData)
+        {
+            // Show the startup object
+            if (StartupObject != null)
+            {
+                StartupObject.SetActive(true);
+
+                if (MoveCollectionOnDismiss)
+                {
+                    // Update the Hologram Collection's position so it shows up
+                    // where the Fitbox left off. Start with the camera's localRotation...
+                    Quaternion camQuat = Camera.main.transform.localRotation;
+
+                    // ... ignore pitch by disabling rotation around the x axis
+                    camQuat.x = 0;
+
+                    // Rotate the vector and factor y back into the position
+                    Vector3 newPosition = camQuat * collectionStartingOffsetFromCamera;
+                    newPosition.y = collectionStartingOffsetFromCamera.y;
+
+                    // Position was "Local Position" so add that to where the camera is now
+                    StartupObject.transform.position = Camera.main.transform.position + newPosition;
+
+                    // Rotate the Hologram Collection to face the user.
+                    Quaternion toQuat = Camera.main.transform.localRotation * StartupObject.transform.rotation;
+                    toQuat.x = 0;
+                    toQuat.z = 0;
+                    StartupObject.transform.rotation = toQuat;
+                }
+            }
+            // Destroy the Fitbox
+            Destroy(gameObject);
```

Comment:
Don't forget to add `eventData.Use();` to consume the input event.

Reply:
ok just did.  do i need to do a new pull request or will this new commit just attach itself?

Reply:
Just push your updates to your branch, that will update the PR automatically.

Reply:
done